db/view/view_building_worker.cc

```diff
@@ -851,6 +672,116 @@ future<> view_building_worker::do_process_staging(table_id table_id, dht::token
     co_await _vug.process_staging_sstables(std::move(table), std::move(sstables));
 }
 
+future<view_building_state> view_building_worker::get_latest_view_building_state() {
+    return smp::submit_to(0, [&sharded_vbw = container()] {
+        auto& vbw = sharded_vbw.local();
+        auto guard = vbw._group0_client.start_operation(vbw._as);
+        return vbw._vb_state_machine.building_state;
```

Comment:
`start_operation` returns a future, so it should be co-awaited.

I see that this code is ~removed~ commented out in further commits. It would be good not to have the wrong code in the first place, though.